modules/install/pages/install.adoc

```diff
@@ -1,71 +1,538 @@
+=========================================================================
+  JBoss Bootstrap Environment
+  JBOSS_HOME: /opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8
+  JAVA: java
+_...output omitted..._
+05:56:12,465 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0051: Admin console listening on http://127.0.0.1:9990
+05:56:12,488 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: JBoss EAP 8.0 Update 1.0 (WildFly Core 21.0.5.Final-redhat-00001) started in 3561ms - Started 282 of 524 services (318 services are lazy, passive or on-demand) - Server configuration file in use: standalone.xml
+----
 
-* Links to all relevant product docs
+* Navigate to the JBoss EAP console at `http://localhost:8080` using firefox browser and confirm JBoss EAP is running as shown in the below screenshot.
++
+image::ui.png[align="center"]
 
-== Lab 1: Installing JBoss EAP using the Zip file
+[NOTE]
+--
+The jboss-eap-installation-manager is also supported on Microsoft Windows. To use this script on a Windows machine, replace the `.sh` with `.bat` in your script.
+--
+
+* Shutdown the JBoss EAP 8 server by pressing `Ctrl+C` in the terminal window where you started JBoss EAP 8.
+
+* To uninstall the JBoss EAP 8 server, simple remove the `/opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001` directory.
+
+== Lab 2: Installing JBoss EAP using archive method
+
+*Outcome*: In this lab, you download `jboss-eap-8.0.0.zip` archive file and install JBoss EAP as a standalone server that can be access at `http://localhost:8080`.
+
+* Take GUI access of RHEL 9 server.
+
+* Download the `jboss-eap-8.0.0.zip` by following below steps:
+** Open a firefox browser.
+** Log in to the https://access.redhat.com/[Red Hat Customer Portal].
+** Click Downloads.
+** Select `Red Hat JBoss Enterprise Application Platform` from the `Product Downloads` list.
+** In the Version drop-down list, select `8.0`.
+** Find `Red Hat JBoss Enterprise Application Platform 8.0.0` in the list and click the `Download` link.
+** The file would get downloaded under your `~/Downloads` directory.
+
+* Open a terminal instance and copy the `jboss-eap-8.0.0.zip` file present under `~/Downloads` directory to `/opt` directory as a sudo user.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo cp ~/Downloads/jboss-eap-8.0.0.zip /opt
+----
+
+* Swtich to `/opt` directory and uncompress the `jboss-eap-8.0.0.zip` file as a sudo user.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ cd /opt
+[vagrant@server opt]$ sudo unzip jboss-eap-8.0.0.zip
+----
+
+* Switch to the `bin` directory present under the uncompressed `jboss-eap-8.0.0.zip` file.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server opt]$ cd jboss-eap-8.0/bin/
+[vagrant@server bin]$
+----
+
+* Install JBoss EAP by executing the `./jboss-eap-installation-manager.sh` script as a sudo user.
```

Comment:
why is sudo needed? I thought we agreed to run EAP as a non-root normal user called jboss? are you missing chmod commands somewhere in the hands on lab?